src/thrift-client/TgtInterfaceImpl.cpp

```diff
@@ -828,19 +910,33 @@ void StordVmdk::ScheduleNow(folly::EventBase* basep, StorRpcAsyncClient* clientp
 
 	basep->runInEventBaseThread([this, clientp, basep,
 			pending = std::move(pending)] () {
+		std::unique_ptr<std::vector<::hyc_thrift::WriteRequest>> write;
+		uint32_t nwrites = 0;
+
 		for (auto reqp : pending) {
 			switch (reqp->type) {
 			case Request::Type::kRead:
 				ScheduleRead(basep, clientp, reqp);
 				break;
-			case Request::Type::kWrite:
```

Comment:
Prasad, do the lower layers have support for normal writes as well as bulk write, or only for bulk writes now. Saying that we may want to keep the support for normal writes intact so that in future if required we can toggle between normal writes and bulk variant.

Reply:
The write support remains as it is.